redbot/core/utils/mod.py

```diff
@@ -12,6 +12,7 @@
 async def mass_purge(
     messages: List[discord.Message],
     channel: Union[discord.TextChannel, discord.VoiceChannel, discord.Thread],
+    reason: str = None,
```

Comment:
The linked issue specified this should be a kwarg, and I do think it would be better as one.

Reply:
Would `**reason: str,` be the correct way to specify this as a kwarg here?

Reply:
To make something a keyword-only argument, it needs to be specified after `*` symbol in the argument list, e.g:
```suggestion
    *,
    reason: str = None,
```